docs/developer-docs/integrations/ethereum/evm-rpc.md

```diff
@@ -0,0 +1,365 @@
+# EVM RPC canister
+
+To use the EVM RPC canister, you can import it into your project's `dfx.json` file:
+
+```json
+{
+ "canisters": {
+   "evm_rpc": {
+     "type": "custom",
+     "candid": "https://github.com/internet-computer-protocol/ic-eth-rpc/releases/latest/download/evm_rpc.did",
+     "wasm": "https://github.com/internet-computer-protocol/ic-eth-rpc/releases/latest/download/evm_rpc_dev.wasm.gz",
+     "remote": {
+       "id": {
+         "ic": "a6d44-nyaaa-aaaap-abp7q-cai"
+       }
+     }
+   }
+ }
+}
+```
+
+## Supported RPC methods
+
+- `eth_gasPrice`: Queries the current ETH gas price.
+- `eth_getLogs`: Queries the logs of a specified block or transaction.
+- `eth_getBlockByNumber`: Queries information about a given block.
+- `eth_getTransactionReceipt`: Queries details about a submitted transaction.
+- `eth_feeHistory`: Queries the historical fee data to estimate gas prices for transactions.
+- `eth_sendRawTransaction`: Submits a signed transaction to the Ethereum network.
+- `eth_getTransactionCount`: Queries the number of transactions for a specified address.
+- `eth_call`: Calls a smart contract function without changing the blockchain's state.
+- `eth_getStorageAt`: Queries the value of a storage position at a given address.
+- `eth_estimateGas`: Provides an estimate for the gas consumption of a transaction before it is mined.
+- `eth_blockNumber`: Queries the number of the most recent block.
+- `eth_getCode`: Queries the bytecode of a specified contract.
```

Comment:
We currently only have first-class support for six of these methods, while the rest can be accessed with the `request` method. Here's a suggestion to give a sense for the distinction:

```suggestion
- `eth_feeHistory`: Queries the historical fee data to estimate gas prices for transactions.
- `eth_getLogs`: Queries the logs of a specified block or transaction.
- `eth_getBlockByNumber`: Queries information about a given block.
- `eth_getTransactionCount`: Queries the number of transactions for a specified address.
- `eth_getTransactionReceipt`: Queries details about a submitted transaction.
- `eth_sendRawTransaction`: Submits a signed transaction to the Ethereum network.

Other JSON-RPC methods (including those specific to non-Ethereum networks) may be accessed using the canister's `request` method.
```